CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.21.0-rc.0 / 2020-08-26
+
+* [CHANGE] Promtool: Add tsdb commands as promtool tsdb subcommand. #6088
+* [FEATURE] Docker Swarm SD: Support tasks and service without published ports. #7686
+* [FEATURE] Eureka SD: New service discovery. #3369
+* [FEATURE] Hetzner SD: New service discovery. #7822
+* [FEATURE] Kubernetes SD: Support Kubernetes EndpointSlices. #6838
+* [ENHANCEMENT] Support composite durations in PromQL, config and UI, e.g. 1h30m. #7713 #7833
+* [ENHANCEMENT] DNS SD: Add SRV record target and port meta labels. #7678
+* [ENHANCEMENT] PromQL: Reduce the amount of data queried by remote read when a subquery has an offset. #7667
+* [ENHANCEMENT] Promtool: add --time option to query instant command. #7829
+* [ENHANCEMENT] Rules: Label rule_group_iterations metric with group name. #7823
+* [ENHANCEMENT] Scrape: Add per scrape-config targets limit. #7554
+* [ENHANCEMENT] Scrape: Do not stop scrapes in progress during reload. #7752
```

Comment:
Bugfix, as this was already meant to be the behaviour.

Reply:
Yes but it was never implemented or documented, so I'm wondering

Reply:
This isn't the sort of detail that'd be in the user docs, we have told users that reloads are perfectly safe though.